### PR TITLE
example: Add async example of GATT interface

### PR DESCRIPTION
**Pull request by kawasaki**
Hello @igo95862,

Recently, I wrote a simple python script and confirmed that python-sdbus-bluez works with BLE (Bluetooth Low Energy) device as expected. I suggest to add the script to python-sdbus-bluez  as another example. Could you take a look in it?

### CODE REVIEW

**Comment by igo95862** [comment]
Hello @kawasaki 

Thank you for submitting pull request.

Let me review the code.

File: examples/sdbus_async/gatt_props.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        if services:
+            for service in services:
+                await print_service_props(dbus, service)
+            break
+            await sleep(1)
+
+    # Clean up
+    await device.disconnect()
+
+run(main())
```
Comment (igo95862) [inline examples/sdbus_async/gatt_props.py]: Do you mind putting this function call under `if __name__ == "__main__":`. It is considered a good practice. (also the `if len(sys.argv) < 2:`)

File: examples/sdbus_async/gatt_props.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+from __future__ import annotations
+
+import sys
+import xml.etree.ElementTree as ET
+from asyncio import run, sleep
+from typing import Any
+
+from sdbus import SdBus, sd_bus_open_system
+from sdbus.dbus_proxy_async_interfaces import DbusIntrospectableAsync
```
Comment (igo95862) [inline examples/sdbus_async/gatt_props.py]: By the way the `DbusInterfaceCommonAsync` inherits from `DbusIntrospectableAsync` and does not need to be imported from private internal module.

**Comment by kawasaki** [comment]
Thanks for the review! Will rework the code next weekend.

**Comment by kawasaki** [comment]
I've reflected your comments and also avoided two global variables, DEV_ADDR and DEV_PATH. Thanks again for the comments. It looks better :)

File: examples/sdbus_async/gatt_props.py [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    # Get services and characteristics and print their properties
+    await sleep(2)
+    for i in range(3):
+        print("Find services...")
+        services = await find_service(dbus, dev_path)
+        if services:
+            for service in services:
+                await print_service_props(dbus, dev_path, service)
+            break
+            await sleep(1)
```
Comment (igo95862) [inline examples/sdbus_async/gatt_props.py:183]: Isn't this an unreachable code because of `break`?

**Comment by igo95862** [comment]
Thank you!

File: examples/sdbus_async/gatt_props.py (hunk 3 above)
Comment (kawasaki) [reply examples/sdbus_async/gatt_props.py:183]: My bad, the next line "await sleep(1)" has extra indent... Will create another pull request to fix it.

**Pull request merged**